projects/capybara-exchange/index.js

```diff
@@ -0,0 +1,24 @@
+const config = require("./config");
+const { sumTokens2 } = require("../helper/unwrapLPs");
+
+Object.keys(config).forEach((chain) => {
+  const arg = config[chain];
+  module.exports[chain] = {
+    tvl: async (_, _b, { [chain]: block }, { api }) => {
```

Comment:
mostly good, change this line to `tvl: async (api ) => {`

Reply:
Thanks! I followed your suggestion. Can you take a look again?